content/en/docs/faq/terminology.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "TLS Terminology"
+linkTitle: "TLS Terminology"
+weight: 50
+type: "docs"
+---
+
+With TLS being so widely deployed, terminology can sometimes get confused or be used to mean different things, and that reality
+combined with the complexity of TLS can lead to serious misunderstandings and confusion.
+
+### What does "publicly trusted" mean?
+
+Broadly speaking, a "publicly trusted" certificate is one that you can use on the public internet and expect
+that most reasonably up-to-date computers will be able to verify it using their system trust store.
```

Comment:
```suggestion
Broadly speaking, a "publicly trusted" certificate is one that you can use on the Internet and expect
that most reasonably up-to-date computers will be able to verify it using their system trust store.
```

Reply:
haha I'd not really thought about it but I guess "public internet" might be a bit redundant! thanks!